controllers/rider.js

```diff
@@ -215,6 +215,36 @@ const psGetPTSalaryInfo = new PS({
     `
 })
 
+const psGetRiderRatingInfo = new PS({
+    name:"rider-rating-info",
+    text: "SELECT uid, name, count(*), TRUNC(avg(value), 1) as avg_rating, date_part('year', date) as year, date_part('month', date) as month " +
+    "FROM ratings JOIN orders using(oid) JOIN riders on (riderid = uid) JOIN Users using(uid) " +
```

Comment:
In this query, those without ratings would not be shown instead of showing a rider with zero rating and zero count.

Reply:
I thinks thats the intended behaviour

Reply:
hmm i think we sohuld include riders with zero ratings and zero count

Reply:
ok i'll add it